crt_portal/static/sass/custom/landing.scss

```diff
@@ -226,6 +239,14 @@ $arrow-height: 24px;
     margin-bottom: 2.5rem;
   }
 
+  @include at-media-max(mobile-lg) {
+    .crt-landing--description__rights {
+      @include text--heading__large();
+      margin-top: 1rem;
+      margin-bottom: 2rem;
+    }
+  }
```

Comment:
Rather than duplicate the text size and margin-top rules here, move the media query up to line 239 into the existing `.crt-landing--description__rights` rule. So lines 236 and onwards would look like this (similar to what you did on line 267):
```
.crt-landing--description__rights {
  @include text--heading__large();
  margin-top: 1rem;
  margin-bottom: 2.5rem;
  @include at-media-max(mobile-lg) {
    margin-bottom: 2rem;
  }
}
```